src/components/ContentWidgets/Homepage/Homepage.css

```diff
@@ -0,0 +1,14 @@
+* {
```

Comment:
You should not override global styles for this. You should wrap your component in some `.homepage` class and then use it within your CSS.